handlers/main.yml

```diff
@@ -2,6 +2,16 @@
 # file: handlers/main.yml
 #
 
+- name: reload systemd prometheus
```

Comment:
My suggestion would be to remove `reload systemd prometheus`, `reload systemd alertmanager` and use this instead:

```yaml
- name: reload systemd
  systemd:
    daemon_reload: yes
```

Reply:
I actually don't think you can do `daemon_reload` without a service